src/main/resources/application.yml

```diff
@@ -4,6 +4,11 @@ spring:
   profiles:
     active: default
 
+security:
+  api:
+    key: ${API_KEY:e8-Mm0ybormRil7k_DZO9jYtRAYW5VX5MCQiQG2CLD4}
```

Comment:
this change does not work, actually we should override like this via deployment: 
SECURITY_API_KEY: "key"

Reply:
```suggestion
    key: e8-Mm0ybormRil7k_DZO9jYtRAYW5VX5MCQiQG2CLD4
```